quiz.py

```diff
@@ -3,4 +3,18 @@
 Author: Name
 Date: YYYY-MM-DD
 Description: A brief explanation of what this program does.
-"""
\ No newline at end of file
+"""
+print ("Multiple-Choice Quiz Game")
+print ("")
+print ("1. Which of these agents role is a Duelist?")
+iso = ("(a) Iso")
+kj = ("(b) Killjoy")
+omen = ("(c) Omen")
+print (iso)
+print (kj)
+print (omen)
+a = input("> ")
```

Comment:
You're using a variable here for user input that, presumably, will be checked for correctness, then assigned again for the next question. This changing condition is proper use of a variable, unlike my comment above. Make sure to give the variable name something sensible, though, not just `a`